File: src/main/java/net/greenjab/fixedminecraft/util/ExhaustionHelper.java
package net.greenjab.fixedminecraft.util;

import net.minecraft.entity.player.PlayerEntity;

public class ExhaustionHelper
{
    public interface ExhaustionManipulator
    {
        float getExhaustion();

        void setExhaustion(float exhaustion);
    }

    public static float getExhaustion(PlayerEntity player)
    {
        return 0;
        //return ((ExhaustionManipulator) player.getHungerManager()).getExhaustion();
    }

    public static void setExhaustion(PlayerEntity player, float exhaustion)
    {
        //((ExhaustionManipulator) player.getHungerManager()).setExhaustion(exhaustion);
    }
}

File: src/main/kotlin/net/greenjab/fixedminecraft/util/ExhaustionHelper.kt
package net.greenjab.fixedminecraft.util

import net.minecraft.entity.player.PlayerEntity

object ExhaustionHelper {
    fun getExhaustion(player: PlayerEntity): Float {
        return (player.hungerManager as ExhaustionManipulator).exhaustion
    }

    fun setExhaustion(player: PlayerEntity, exhaustion: Float) {
        (player.hungerManager as ExhaustionManipulator).exhaustion =
            exhaustion
    }

    interface ExhaustionManipulator {
        var exhaustion: Float
    }
}
